packages/dmn-js-drd/src/features/modeling/DrdUpdater.js

```diff
@@ -151,6 +151,10 @@ export default function DrdUpdater(
         newTarget = context.newTarget,
         newTargetBo = newTarget.businessObject;
 
+    if (is(connectionBo, 'dmn:Association')) {
```

Comment:
So association parents never change?

Reply:
Semantic parents don't. `dmn:Association` and `dmn:TextAnnotation` elements are always found in `dmn:Definitions#artifacts` - unlike e.g. `dmn:InformationRequirement` which is found in its target.

Reply:
Thanks. Looks good to me then.